hedera-node/src/main/java/com/hedera/services/legacy/handler/FreezeHandler.java

```diff
@@ -182,7 +183,11 @@ private void updateFeatureWithFileContents(final byte[] fileBytes) {
 			if (directory.exists()) {
 				log.info("{} clean directory {}", LOG_PREFIX, directory);
 				// delete everything in it recursively
-				FileUtils.cleanDirectory(directory);
+				try (final var walk = Files.walk(directory.toPath())) {
+					walk.sorted(Comparator.reverseOrder())
+							.map(Path::toFile)
+							.forEach(File::delete);
+				}
```

Comment:
Is there an advantage of this implementation over `FileUtils.cleanDirectory`?

Reply:
I think `FileUtils.cleanDirectory` doesn't delete the folder itself. `FileUtils.deleteDirectory` probably would. To me the `Files.walk` here shows clearly the same behavior as an `rm -rf directory` would do, while we have to rely on the correctness of the helper FileUtils function which we can misuse.